freqtrade/plugins/pairlist/AgeFilter.py

```diff
@@ -49,7 +56,9 @@ def short_desc(self) -> str:
         Short whitelist method description - used for startup-messages
         """
         return (f"{self.name} - Filtering pairs with age less than "
-                f"{self._min_days_listed} {plural(self._min_days_listed, 'day')}.")
+                f"{self._min_days_listed} {plural(self._min_days_listed, 'day')}"
+                " or more than "
+                f"{self._max_days_listed} {plural(self._max_days_listed, 'day')}")
```

Comment:
I think this message will need to be composed dynamically.

if `max_days_listed` is not set - it'll result in "or more than None days" (or an exception ...).